### PR TITLE
docs(expansion-panel)

### PR DESCRIPTION
**Pull request by Vanessarich**

### CODE REVIEW

**Comment by matmkian** [comment]
En attente de finalisation de définition.

File: demo/angularjs/components/expansion-panel/doc.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Drag option
+
+A drag handle can be added.
+
+<demo-block component="expansion-panel" partial="drag"></demo-block>
+
+## Trimmed
+
+Use trimmed when you don't need the left and right spacing.
```
Comment (lionelmoi) [inline demo/angularjs/components/expansion-panel/doc.md]: if `trimmed`is a named option it must be capitalized
Do not use contraction.
I do not understand this sentence -> which spacing? the margin? the padding?

File: demo/angularjs/components/expansion-panel/doc.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Expansion panel
+
+**Expansion panels deliver large amounts of content in a small space through <a href="https://en.wikipedia.org/wiki/Progressive_disclosure" target="_blank">progressive disclosure</a>.**
+
+## Default
+
+<demo-block component="expansion-panel" partial="default"></demo-block>
+
+## Drag option
```
Comment (lionelmoi) [inline demo/angularjs/components/expansion-panel/doc.md]: Why is the title not `Drag handle`?

File: demo/angularjs/components/expansion-panel/doc.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+A drag handle can be added.
```
Comment (lionelmoi) [inline demo/angularjs/components/expansion-panel/doc.md]: Explain how the drag handle behaves. Is it always horizontal or vertical or both? Why should developers use it?

File: demo/angularjs/components/expansion-panel/doc.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Expansion panel
+
+**Expansion panels deliver large amounts of content in a small space through <a href="https://en.wikipedia.org/wiki/Progressive_disclosure" target="_blank">progressive disclosure</a>.**
```
Comment (lionelmoi) [inline demo/angularjs/components/expansion-panel/doc.md]: I would say Expansion panels `implement` a progressive disclosure pattern.

**Pull request merged**